keps/sig-node/2535-ensure-secret-pulled-images/README.md

```diff
@@ -179,10 +178,27 @@ or expect preloaded images since boot.
 
 ## Design Details
 
-Kubelet will track, in memory, a hash map for the credentials that were successfully used to pull an image. The hash map
-will not be persisted to disk, in alpha. For alpha explicitly, we will not reuse or add other state manager concepts to kubelet.
+Kubelet will track, in memory, a hash map for the credentials that were successfully used to pull an image. It has been decided that the hash map will be persisted to disk, in alpha.
```

Comment:
What is the max size of this cache?  If images are frequently changed, does it grow without bounds?

Reply:
we have decided to enable garbage collection for the alpha as a part of adding persistence updating the kep...

Reply:
max size would be the number of unique cached entries * the number of unique images that have not been garbage collected by kubelet

Reply:
In the pr, we will do a cleanup for the cache file every minute(or hour) if the cache is out-of-date, by default 1 hour.

The disk cache is mainly for restarting kubelet or node reboot.

Reply:
I would clarify this in the KEP.

Reply:
should be resolved